src/controllers/OauthClientsController.js

```diff
@@ -8,22 +8,22 @@ import route from '../decorators/route';
 class OauthClientsController extends Controller {
   @httpVerb('post')
   @route('/v1/products/:productIDorSlug/clients/')
-  // eslint-disable-next-line class-methods-use-this
-  async createClient(): Promise<*> {
```

Comment:
yeah. I think `class-methods-use-this` is a useless rule, may just disable it in eslint-config?